### ISSUE TITLE
Initializing an empty map not working

### ISSUE THREAD

**Issue by Plex768** [angular-ui/angular-google-maps#1780]
I'm trying to initialize an empty google map when the app starts.  This should happen before the app tries to get the users current geolocation or even if the user turned off wi-fi and the 4G data connection.  I have the code below but it is not working. It just displays a blank white screen. Any help would be greatly appreciated.  Thanks.

```
.controller("Example", function($scope, $ionicPlatform) {

$ionicPlatform.ready(function() {

 $scope.map = {
                            center: {
                            latitude: '',
                            longitude: ''
                        },
                        zoom: 16
                    };

});
 .... (and so on)
});
```

**Comment by ZachMoreno** [comment]
Good Morning @Plex768,

I'm happy to help you achieve your desired outcome, but I need a little more information to work with. To clarify, Angular Google Maps & the vanilla Google Maps JS API both require a network connection to load all Google Map assets (tiles, libraries, etc.).

1. Is the issue occurring when testing in your browser, mobile device emulator, or on a physical mobile device?
2. I believe your map will not display without integer `map.center.latitude` & `map.center.longitude` values. Have you tried the above code with lat/long values instead of empty strings?
2. Can you replicate the issue in a [JSFiddle](https://jsfiddle.net/), [CodePen](http://codepen.io/), [JSBin](https://jsbin.com/?html,js,output)?

Try the snippet below, which will center over California.

```
$scope.map= {
	center: {
		latitude: 37.632711,
		longitude: -120.572511
	},
	zoom: 6
};
```

If the above snippet resolves your issue, please let me know & I'll happily request that this issue be closed by the maintainer.

**Comment by Plex768** [comment]
Hi,

Thank you for your reply. I have seen other android apps including native android apps where it is able to render a map on the screen even if both the data and wi-fi connections are turned off.  

Also, using your snippet above works but what I want to do is this. I have a real time bus location app where users will see real time geolocation of buses on the map.  the bus routes will be highlighted in color.  so, even if the user does not have any data connection i want the app to load the map and show the routes in color. I will cache the route information locally on the phone so that the app does not have to fetch the info from a server and if there is no data connection, then it should say that in a toast message after loading the map.  Of course, the map should not show the current location marker if the phone does not have an active data connection. Is this possible?

Thanks.

**Comment by ZachMoreno** [comment]
No worries @Plex768,

It is my understanding that the Google Maps JS API, & therefor Angular Google Maps, does not support offline. In a hybrid app like you're building it is possible to utilize the Google Maps Android API to support offline, but that is out of scope for Angular Google Maps. Is my understanding correct @nmccready?

**Comment by Plex768** [comment]
OK I understand.  Thank you for your help.

**Comment by ZachMoreno** [comment]
@Plex768 

No worries, happy to help & sorry this isn't the project you're looking for.
Can you please mark this issue as closed?

**Comment by Plex768** [comment]
Actually, this ionic I think can render maps even when offline because it uses the native google maps SDK?  https://market.ionic.io/starters/google-maps

I tried to follow their directions to run build and run on my device but the map is not showing. I just see a blank white screen.

**Comment by Plex768** [comment]
https://github.com/mapsplugin/cordova-plugin-googlemaps this is the cordova google maps plugin that is a wrapper for the Google Maps Android SDK v2 and Google Maps SDK for iOS. Would I be able to do what I want with this plugin? 

thanks.

**Comment by ZachMoreno** [comment]
If those projects support offline, then yes I suppose, but I don't have experience with them myself.

**Comment by Plex768** [comment]
OK thanks.

**Comment by nmccready** [comment]
@ZachMoreno thanks for your help and this is out of scope. Unless of-course someone else wants to champion it.
